CMakeLists.txt

```diff
@@ -129,7 +129,7 @@ endif()
 
 # Set up ACTSVG for displaying
 option( DETRAY_SETUP_ACTSVG
-   "Set up the actsvg target(s) explicitly" ${DETRAY_SVG_DISPLAY} )
+   "Set up the actsvg target(s) explicitly" ${DETRAY_BUILD_TEST_UTILS} )
```

Comment:
What happens if anyone requests ```DETRAY_SVG_DISPLAY=ON```? Because that will not turn test utils on...

Reply:
OK I changed it back